### PR TITLE
[Rebase & FF] Enabled pipelines and CodeQL for PolicyServicePkg

### PR DESCRIPTION
**Pull request by cfernald**
## Description

Enables the pipelines for the policy service after the 202405 rebase.

- [ ] Impacts functionality?
- [ ] Impacts security?
- [ ] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?

## How This Was Tested

N/A

## Integration Instructions

N/A

### CODE REVIEW

File: .azurepipelines/Ubuntu-GCC5.yml [hunk 1]
```diff
@@ -50,8 +50,8 @@ extends:
         Pkgs: 'PcAtChipsetPkg,ShellPkg,StandaloneMmPkg,BaseTools'
         Targets: 'DEBUG,RELEASE,NO-TARGET,NOOPT'
         ArchList: $(arch_list)
-      TARGET_TEST:
-        Pkgs: 'UnitTestFrameworkPkg'
+      TARGET_TEST_POLICY:
```
Comment (os-d) [inline .azurepipelines/Ubuntu-GCC5.yml:53]: This needs to be added to the Windows yml, too, right?

**Review by os-d — COMMENTED** [review]
Can you add [SQUASH ON REBASE] to the PR title? And then in the description link to the commit that this should be squashed with (adding PolicyServicePkg).

**Review by os-d — APPROVED** [review]

File: .azurepipelines/Ubuntu-GCC5.yml (hunk 1 above)
Comment (cfernald) [reply .azurepipelines/Ubuntu-GCC5.yml:53]: oops

Comment (cfernald) [reply .azurepipelines/Ubuntu-GCC5.yml:53]: funnily, we aren;t building UnitTestFrameworkPkg on windows for some reason..

**Comment by cfernald** [comment]
> Can you add [SQUASH ON REBASE] to the PR title? And then in the description link to the commit that this should be squashed with (adding PolicyServicePkg).

I will probably split into two commits as we are also missing unit test pkg on windows for some reason, so but will do this in their commit messages

**Review by apop5 — APPROVED** [review]

File: .azurepipelines/Windows-VS.yml [hunk 2]
```diff
@@ -49,6 +49,10 @@ extends:
         Pkgs: 'PcAtChipsetPkg,ShellPkg,StandaloneMmPkg,BaseTools'
         Targets: 'DEBUG,RELEASE,NO-TARGET,NOOPT'
         ArchList: $(arch_list)
+      TARGET_TEST:
+        Pkgs: 'UnitTestFrameworkPkg'
```
Comment (makubacki) [inline .azurepipelines/Windows-VS.yml]: UnitTestFrameworkPkg is on Line 85.

Comment (makubacki) [reply .azurepipelines/Windows-VS.yml]: I see, that's only the self-host build...

**Review by makubacki — APPROVED** [review]

**Pull request merged**